source/Calamari/Scripts/Octopus.Features.IISWebSite_BeforePostDeploy.ps1

```diff
@@ -436,6 +436,11 @@ if ($deployAsWebSite)
 	}
 
 	ForEach($binding in $bindingArray){
+		if(![Bool]::Parse($binding.enabled)) {
+    		Write-IISBinding "Ignore binding: " $binding
+    		return
```

Comment:
```suggestion
    		continue
```

Reply:
I've just pushed this up and am running a build locally to test but I'm pretty confused here. I tested this in a basic script with 

```
1..3 | ForEach-Object  {
    $binding = New-Object PSObject
    $binding | Add-Member -MemberType NoteProperty -Name "enabled" -Value $false
    Write-Host "With Continue"

    if(![Bool]::Parse($binding.enabled)) {
        Write-Host "disabled"
    	continue
    }
}
```

and get 
![image](https://github.com/OctopusDeploy/Calamari/assets/101079287/1dd985b0-4082-49c7-8f6d-789e06f458e8)

Switching to return I get 
![image](https://github.com/OctopusDeploy/Calamari/assets/101079287/7402ecbb-01b8-4676-9a0d-a21d38f81668)

Running the script itself with alternating disabled toggles it looks like your right though...
![image](https://github.com/OctopusDeploy/Calamari/assets/101079287/541e89dd-a91a-4d5d-b28e-4d319188ec54)

Do you know why that above example is incorrect?

Reply:
Ahh https://stackoverflow.com/questions/7760013/why-does-continue-behave-like-break-in-a-foreach-object 😥 now I'm just confused why Powershell made this different on a ForEach vs a ForEach-Object. Thanks for the catch

Reply:
sigh...`return` it is.

Reply:
I think it's return in `ForEach-Object` but continue in `ForEach` which is this case. Because Cmdlet vs Loop... Stoopid Powershell

Reply:
ah, sorry. I'm blind, didn't realise your test script was using `ForEach-Object` instead of `ForEach` as in the pre-postdeploy script in Calamari.

Reply:
![image](https://github.com/OctopusDeploy/Calamari/assets/101079287/e0853fd8-a00b-40bf-9f3a-71b306ac28aa)
![image](https://github.com/OctopusDeploy/Calamari/assets/101079287/26f198c6-95a3-4329-a7c5-c3fda52dc93f)